### PR TITLE
Readme update - spelling and grammar fixes

### PR DESCRIPTION
**Pull request by sethbergman**
**Description of the change**
Made some modifications to the README. I corrected some spelling and grammar and made it easier to copy the commands inside backticks.

**Benefits**
They are minor, but make your README appear more professional.

**Possible drawbacks**
None. I merged the latest changes to master.

### CODE REVIEW

**Comment by sethbergman** [comment]
I've updated this PR and fixed the merge conflicts. It's ready for review @sameersbn. Thanks! :rocket:

File: README.md [hunk 1]
```diff
@@ -120,19 +120,19 @@ If you want to run the application manually instead of using docker-compose, the
     bitnami/ghost:latest
   ```
 
-Access your application at http://your-ip/
+Access your application at http://your-ip/.
 
-> **Note!** If you want to access your application from a public IP or hostname you need to properly configured Ghost . You can handle it adjusting the configuration of the instance by setting the environment variable `GHOST_HOST` to your public IP or hostname.
+> **Note!** If you want to access your application from a public IP or hostname you need to properly configure your Ghost instance. You can handle it by adjusting the configuration of the environment variable `GHOST_HOST` to your public IP or hostname.
```
Comment (marcosbc) [inline README.md]: Does it make sense to change "configuration" to "value" for you?

File: README.md [hunk 2]
```diff
@@ -120,19 +120,19 @@ If you want to run the application manually instead of using docker-compose, the
...
 
 ## Persisting your application
 
-If you remove the container all your data and configurations will be lost, and the next time you run the image the database will be reinitialized. To avoid this loss of data, you should mount a volume that will persist even after the container is removed.
+If you remove the container, all of your data and configuration will be lost. The next time you run the image, the database will be reinitialized. To avoid this loss of data you should mount a volume that will persist, even after the container is removed.
```
Comment (marcosbc) [inline README.md]: We should be using plural forms in the first sentence: "and configuration**s**"

File: README.md [hunk 3]
```diff
@@ -203,35 +203,35 @@ In this case you need to specify the directories to mount on the run command. Th
 
 # Upgrade this application
 
-Bitnami provides up-to-date versions of MariaDB and Ghost, including security patches, soon after they are made upstream. We recommend that you follow these steps to upgrade your container. We will cover here the upgrade of the Ghost container. For the MariaDB upgrade see https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#upgrade-this-image
+Bitnami provides up-to-date versions of MariaDB and Ghost, including security patches as soon as they are made upstream. We recommend that you follow these steps to upgrade your container. We will cover the upgrade of the Ghost container. For the MariaDB upgrade see https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#upgrade-this-image
```
Comment (marcosbc) [inline README.md]: I believe we should change the following sentence, from this:

> including security patches as soon as they are made upstream.

To this:

> including security patches soon after they are made available upstream.

File: README.md [hunk 4]
```diff
@@ -341,29 +341,29 @@ This would be an example of SMTP configuration using a GMail account:
...
+We'd love for you to contribute to this container. You can request new features by creating a [feature request](https://github.com/bitnami/bitnami-docker-ghost/issues), or submit a [pull request](https://github.com/bitnami/bitnami-docker-ghost/pulls) with your contribution.
 
 # Issues
 
-If you encountered a problem running this container, you can file an [issue](https://github.com/bitnami/bitnami-docker-ghost/issues). For us to provide better support, be sure to include the following information in your issue:
+If you've encountered a problem running this container, you can file an [issue here](https://github.com/bitnami/bitnami-docker-ghost/issues). For us to provide better support, be sure to include the following information in your issue:
 
 - Host OS and version
-- Docker version (`docker version`)
+- Docker version `docker version`
```
Comment (marcosbc) [inline README.md]: In my opinion, the previous was correct. You could also leave it like this:

- Docker version: `docker version`

File: README.md [hunk 5]
```diff
@@ -341,29 +341,29 @@ This would be an example of SMTP configuration using a GMail account:
...
 - Output of `docker info`
-- Version of this container (`echo $BITNAMI_IMAGE_VERSION` inside the container)
-- The command you used to run the container, and any relevant output you saw (masking any sensitive information)
+- Version of this container `echo $BITNAMI_IMAGE_VERSION` (inside the container)
```
Comment (marcosbc) [inline README.md]: Same as above, I think the previous was correct.

File: README.md [hunk 6]
```diff
@@ -341,29 +341,29 @@ This would be an example of SMTP configuration using a GMail account:
...
+- The command you used to run the container and any relevant output. Don't forget to mask any sensitive information.
 
-# License
+# [License](#LICENSE)
```
Comment (marcosbc) [inline README.md]: Keep in mind that, by default, GitHub already adds an anchor to each section, so the following link works properly (and scrolls to the license section): https://github.com/bitnami/bitnami-docker-ghost/#license

Could you revert this change?

File: README.md [hunk 7]
```diff
@@ -341,29 +341,29 @@ This would be an example of SMTP configuration using a GMail account:
...
 
 Copyright 2016-2018 Bitnami
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
+Licensed under the Apache License, Version 2.0 (the "License").
```
Comment (marcosbc) [inline README.md]: This is Apache-specific license text. I don't think we should change that. You can find related info here: https://www.apache.org/licenses/LICENSE-2.0#apply

**Comment by sethbergman** [comment]
Thanks for your time while reviewing my PR @marcosbc. These are all great suggestions. I will update the PR with your requests, but it may take me a couple days to find the time. I've been swamped lately. Thank you for your patience, and I appreciate your thorough review! I am very grateful for the guidance. :+1:

File: README.md [hunk 8]
```diff
@@ -2,17 +2,17 @@
...
+https://ghost.org
 
 # TL;DR;
 
 ## Docker Compose
 
 ```bash
 $ curl -sSL https://raw.githubusercontent.com/bitnami/bitnami-docker-ghost/master/docker-compose.yml > docker-compose.yml
-$ docker-compose up -d
+docker-compose up -d
```
Comment (marcosbc) [inline README.md]: It seems you forgot to revert this change.

**Comment by marcosbc** [comment]
If you don't have much time for this and prefer us to take ownership of this PR and make the changes, please let us know and we'll do so.
